packages/legend-query-builder/src/stores/QueryBuilderState.ts

```diff
@@ -102,6 +104,7 @@ export abstract class QueryBuilderState implements CommandRegistrar {
   showFunctionsExplorerPanel = false;
   showParametersPanel = false;
   isEditingWatermark = false;
+  isCheckingEntitlments = false;
```

Comment:
do u need this flag?
I have seen that you have added similar flag in ` QueryBuilderCheckEntitlementsState`